docs/common-reasons-for-ads-not-showing.mdx

```diff
@@ -0,0 +1,63 @@
+# Common reasons for ads not showing
+
+Use the following guide to understand common reasons why apps show few or no ads.
+
+## Did you recently create your AdMob account?
+
+Wait up to 24 hours after you create your account.
+When you first sign up for AdMob, your account is reviewed before it’s approved.
+This typically takes up to 24 hours, but in rare cases can take up to 2 weeks.
+You’ll be notified via email if your account is not approved by AdMob.  
+
+## Is your app or ad unit new?
+
+Wait up to one hour after you create an app or ad unit.
+In some cases, it can take a few days for ads to appear in new apps or ad units.
+Note: New iOS apps will not show Google ads until they’re listed in the Apple App Store.
+Newly registered apps and newly created ad units typically take up to an hour to activate.
+Because of this, you may not see live impressions immediately.
+
+## Test ads not showing up
+
+If you set up an app-ads.txt file for your app, you need to also include this line in your app-ads.txt file in order to load ads using the demo ad units:
```

Comment:
You linked to the upstream app ads documentation for ios and android before, I think it would be great to include the link here.

More broadly - any links to upstream documentation in any of the sections are a positive I think. Documentation is always changing but between the guidance here (which may go out of date, it will probably happen) and links to upstream (which may break, but hopefully not quickly) people should be able to connect the information they need until documentation updates happen. It will add to resiliency and help users be more resourceful I think